iis-workflows/iis-workflows-ingest-pmc/src/main/java/eu/dnetlib/iis/workflows/ingest/pmc/metadata/PmcXmlHandler.java

```diff
@@ -69,8 +70,6 @@
 	
 	public static final String PUB_ID_TYPE_PMID = "pmid";
 	
-	private final Logger log = Logger.getLogger(this.getClass());
-	
 	private Stack<String> parents;
 	
 	private StringBuilder currentValue = new StringBuilder();
```

Comment:
I'm confused why StringBuilder is used for currentValue. I think that normal String is sufficient enough.